src/psyclone/tests/dependency_test.py

```diff
@@ -637,6 +644,28 @@ def test_lfric_stub_args():
     var_accesses = VariablesAccessInfo()
     create_arg_list = KernStubArgList(kernel)
     create_arg_list.generate(var_accesses=var_accesses)
+    var_info = str(var_accesses)
+    assert "field_1_w1: READ+WRITE" in var_info
+    assert "field_2_stencil_dofmap: READ" in var_info
+    assert "field_2_stencil_size: READ" in var_info
+    assert "field_2_w2: READ" in var_info
+    assert "field_3_direction: READ" in var_info
```

Comment:
Again, can this test be extended to check on the scalar/array property of each of these?

Reply:
N/A anymore.